docs/kb/advanced-certificates.md

```diff
@@ -5,53 +5,61 @@ title: VMware Event Broker Appliance - Certificates
 description: Updating Certificates
 permalink: /kb/advanced-certificates
 cta:
- description: With your certificates updated, you can now skip using the `--tls-no-verify` flag while working with faas-cli.
+ description: Replacing the default self-signed TLS certificate in VMware Event Broke Appliance.
 ---
 
 ## Updating the TLS Certificate on VEBA
- 
-The default certificate for OpenFaaS (/ui), Stats (/stats), Status (/status), Logs (/bootstrap) and the other web endpoints running on VEBA are self signed. This might cause browsers to show the certificate as untrusted and would require providing the `--no-tls-verify` flag when working with faas-cli. 
- 
-In order to update the certificates with a certificate from a trusted authority, please follow the steps outlined below
- 
+
+By default, the VMware Event Broker Appliance generates a self-signed TLS certificate that is used to support different web endpoints running on the appliance such as Stats (/stats), Status (/status), Logs (/bootstrap) and Events (/events). This will cause browsers to show the certificate as untrusted.
+
+For organizations that require the use of a TLS certificate from a trusted authority, the VMware Event Broker Appliance provides an option for users to provide their certificate information during the OVF property configuration when deploying the virtual appliance.
+
+In order to use a certificates from a trusted authority, please follow the steps outlined below
```

Comment:
```suggestion
By default, the VMware Event Broker Appliance generates a self-signed TLS certificate that is used to support different web endpoints running on the appliance such as Stats (`/stats`), Status (`/status`), Logs (`/bootstrap`) and Events (`/events`). This will cause browsers to show the certificate as untrusted.

For organizations that require the use of a TLS certificate from a trusted authority, the VMware Event Broker Appliance provides an option for users to provide their certificate information during the OVF property configuration when deploying the virtual appliance.

In order to use a certificates from a trusted authority, please follow the steps outlined below.
```